### PR TITLE
Delete deprecated endpoints lists

### PR DESCRIPTION
**Pull request by mhamra**
|Related issue|
|---|
|#17068|

## Description

Delete deprecated endpoints lists.

### CODE REVIEW

File: framework/wazuh/rbac/decorators.py [hunk 1]
```diff
@@ -66,8 +66,8 @@ def _expand_resource(resource: str) -> set:
             return expand_rules()
         elif resource_type == 'decoder:file':
             return expand_decoders()
-        elif resource_type == 'list:file':
-            return expand_lists()
+        # elif resource_type == 'list:file':
+        #     return expand_lists()
```
Comment (nico-stefani) [inline framework/wazuh/rbac/decorators.py:70]: Why not just remove the condition?

**Review by nico-stefani — CHANGES_REQUESTED** [review]
Please check:
* The PR should be pointing to `11334-dev-new-wazuh-engine`
* The API IT that were failed
* Resolve the conflicts

File: framework/wazuh/rbac/decorators.py (hunk 1 above)
Comment (mhamra) [reply framework/wazuh/rbac/decorators.py]: Removed!

**Review by nico-stefani — CHANGES_REQUESTED** [review]
The PR should point to the new engine epic `11334-dev-new-wazuh-engine`. Also, fix the conflicts, please.

File: framework/wazuh/core/common.py [hunk 2]
```diff
@@ -251,7 +248,7 @@ def get_context_cache() -> dict:
 RULESET_PATH = os.path.join(WAZUH_PATH, 'ruleset')
 RULES_PATH = os.path.join(RULESET_PATH, 'rules')
 DECODERS_PATH = os.path.join(RULESET_PATH, 'decoders')
-LISTS_PATH = os.path.join(RULESET_PATH, 'lists')
-USER_LISTS_PATH = os.path.join(WAZUH_PATH, 'etc', 'lists')
+# LISTS_PATH = os.path.join(RULESET_PATH, 'lists')
+# USER_LISTS_PATH = os.path.join(WAZUH_PATH, 'etc', 'lists')
```
Comment (nico-stefani) [inline framework/wazuh/core/common.py:252]:
```suggestion
```

File: framework/wazuh/rbac/decorators.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 from wazuh.core.common import rbac, broadcast, cluster_nodes
 from wazuh.core.exception import WazuhPermissionError
 from wazuh.core.results import AffectedItemsWazuhResult
-from wazuh.core.utils import expand_rules, expand_lists, expand_decoders
+from wazuh.core.utils import expand_rules, expand_decoders #, expand_lists
```
Comment (nico-stefani) [inline framework/wazuh/rbac/decorators.py:14]:
```suggestion
from wazuh.core.utils import expand_rules, expand_decoders 
```

**Review by nico-stefani — CHANGES_REQUESTED** [review]

**Review by nico-stefani — DISMISSED** [review]
LGTM

**Pull request closed without merging**